src/client/components/DateInput/Calendar.tsx

```diff
@@ -136,10 +136,11 @@ export const Calendar: React.FC<DayzedCalendarProps> = ({
 }) => {
   const [weekdayNamesShort, setWeekdayNamesShort] = React.useState<string[]>([])
   const [monthNamesShort, setMonthNamesShort] = React.useState<string[]>([])
-  const locale = useCurrentLocale()
+  const currentLocale = useCurrentLocale()
+  const currentLocalePath = currentLocale.path
```

Comment:
I've been destructuring the results on the same line. I'm not sure if it's concern if we don't agree on one way of doing tho :)

Reply:
I would vote on a one liner :)

Reply:
Yep, I just think in the case of the `path` it's not a very clear name outside of its context you know? But maybe the solution is rather to change name of that property

Reply:
But of course it's also possible to do `const { path: currentLocalePath } = useCurrentLocale()`

Reply:
Exactly, that's what I tend to do. But I would be up for changing/aliasing the `path` field so something less context dependent!

Reply:
Let's do that in the next PR. Changing name of the `path` property I mean